### PR TITLE
[Docs] Update the return type for load functions

### PR DESCRIPTION
**Pull request by AzizCode92**
fixes this issue #1119.
Before
Screen Shot 2020-12-23 at 19 27 24

After
Screen Shot 2020-12-23 at 19 26 21

### CODE REVIEW

File: torchaudio/backend/_soundfile_backend.py [hunk 1]
```diff
@@ -103,11 +103,15 @@ def load(
...
-            ``[channel, time]`` else ``[time, channel]``.
+    :rtype:
+            tuple(torch.Tensor, int)
+    :returns:
+            torch.Tensor:
+                If the input file has integer wav format and normalization is off, then it has
+                integer type, else ``float32`` type. If ``channels_first=True``, it has
+                ``[channel, time]`` else ``[time, channel]``.
+            int:
+                Sample rate of the audio.
```
Comment (mthrok) [inline torchaudio/backend/_soundfile_backend.py:114]: Could you keep the `Returns:` directive for the consistency with other docstrings?

Something like the following.

```rst
    Returns:
        Tuple[torch.Tensor, int]: Resulting Tensor and sample rate.
             If the input file has integer wav format and normalization is off, then it has
             integer type, else ``float32`` type. If ``channels_first=True``, it has
             ``[channel, time]`` else ``[time, channel]``.
```

**Review by mthrok — COMMENTED** [review]
Looks good but can we keep `Returns:` directive?

File: torchaudio/backend/_soundfile_backend.py (hunk 1 above)
Comment (AzizCode92) [reply torchaudio/backend/_soundfile_backend.py]: Done

File: torchaudio/backend/_soundfile_backend.py [hunk 2]
```diff
@@ -104,10 +104,10 @@ def load(
...
 
     Returns:
-        torch.Tensor:
-            If the input file has integer wav format and normalization is off, then it has
-            integer type, else ``float32`` type. If ``channels_first=True``, it has
-            ``[channel, time]`` else ``[time, channel]``.
+        Tuple[torch.Tensor, int]: Resulting Tensor and sample rate.
+             If the input file has integer wav format and normalization is off, then it has
+             integer type, else ``float32`` type. If ``channels_first=True``, it has
+             ``[channel, time]`` else ``[time, channel]``.
```
Comment (mthrok) [inline torchaudio/backend/_soundfile_backend.py:110]: @AzizCode92  There are stray white spaces introduced.

**Review by mthrok — APPROVED** [review]

**Comment by mthrok** [comment]
Thanks!

**Comment by mthrok** [comment]
If you have time, and if you are interested in, can you cherry-pick this and make a PR against `release/0.7` branch?

**Pull request merged**